src/components/TextAreaField/TextAreaField.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .ui.form textarea:focus,
-.TextArea > textarea:focus {
+.dcl.textArea > textarea:focus {
```

Comment:
uh-oh haha this does have the `.dcl` class, but the casing is camelCase rather than dash-case

Reply:
I changed the TextArea class to text-area and the TextFilter class to text-filter.

Reply:
🙌